masschallenge/opsworks-web-python
# AC-5502 - Provide mechanism for full access to ECS containers

This PR introduces an "install_keys" recipe which installs ssh keys on impact api container instances so that we can all ssh in.

to test this:

- go into an opsworks stack that is configured with an associated impact api server and change the "custom cookbooks" branch to AC-5502 and run the "update custom cookbooks" command.

- find the ip address of the associated impact api server's ec2 instance:
<img width="506" alt="screen shot 2018-04-26 at 3 50 34 pm" src="https://user-images.githubusercontent.com/196425/39328530-a1a38462-4969-11e8-92d0-f9b9034b7ce0.png">

- try to ssh into the machine: ssh ec2-user@<ip address> and note that it fails

- run an "execute custom recipes" command - specify "impact_deploy::install_keys" as the recipe to run.
<img width="396" alt="screen shot 2018-04-26 at 3 48 49 pm" src="https://user-images.githubusercontent.com/196425/39328424-5e1b91d0-4969-11e8-8d33-f9fe29e89c0b.png">

- try to ssh into the machine again and note that it succeeds
